web/src/components/TitlePage.js

```diff
@@ -1,94 +1,83 @@
-import React, { Component } from "react";
-import { withRouter } from "react-router-dom"
-import { Table, Card, CardBody, Form, Input, Button } from "reactstrap";
+import React, {useEffect, useState} from "react";
+import {Button, Card, CardBody, Form, Input, Table} from "reactstrap";
+import {useParams} from "react-router";
 
-class TitlePage extends Component {
-    constructor(props) {
-      super(props);
-      this.state = {
-        data: undefined,
-        ownerChangeValue: "",
-      }
-      this.loadTitle = this.loadTitle.bind(this);
-      this.ownerNameHandleChange = this.ownerNameHandleChange.bind(this);
-      this.ownerNameHandleSubmit = this.ownerNameHandleSubmit.bind(this);
-    }
-    ownerNameHandleChange(event) {
-      this.setState({ownerChangeValue: event.target.value});
+export function TitlePage() {
+
+    let { titleNo } = useParams();
+    const [data, setData] = useState(undefined)
+    const [ownerChangeValue, setOwnerChangeValue] = useState("")
+
+    useEffect( ()=>{
+        loadTitle();
+    }, [])
+
+    useEffect(() =>{
```

Comment:
Let's leave this in here as a banana peel.. but wondering if you see a cleaner way to do this?